docs/app/pages/components/sections/utilities/persistent-data-service/persistent-data-service.component.html

```diff
@@ -0,0 +1,54 @@
+<p>The <code>PersistentDataService</code> can be used to set and get data in <code>localStorage</code>, <code>Cookies</code> or <code>sessionStorage</code>.
+The service can be used to choose which to set or will attempt to use <code>localStorage</code> by default, falling back to <code>Cookies</code> if this fails.</p>
+
+<p>The <code>PersistentDataStorageType</code> enum is used when setting the storage type.</p>
+
+<div class="demo-attributes table-responsive">
+    <table class="table m-t">
+        <tr>
+            <th>Name</th>
+            <th>Args</th>
+            <th>Returns</th>
+            <th>Description</th>
+        </tr>
+        <tr>
+            <td>setItem</td>
+            <td>string, string, PersistentDataStorageType?</td>
+            <td></td>
+            <td>This method expects a key and value parameter as strings and can also accept a 
+                PersistentDataStorageType if you wish to set the storage type, 
+                otherwise it will follow defauly behaviour.</td>
```

Comment:
Typo - 'default' might be good to mention what the default value is also

Reply:
Default behavior is mentioned above this